tests/algorithms/column_pooler_test.py

```diff
@@ -101,7 +101,7 @@ def testLearnSinglePattern(self):
     self.init()
 
     object = self.generateObject(1)
-    self.learn(object, numRepetitions=1, newObject=True)
+    self.learn(object, numRepetitions=2, newObject=True)
```

Comment:
The parameters are:

~~~
      "connectedPermanenceProximal": 0.6,
      "initialDistalPermanence": 0.51,
      "synPermProximalInc": 0.1,
~~~

so each feedforward pattern needs to be seen at least twice before it will reach the `connectedPermanence`. An alternate fix would be to make these tests override these parameters.